docs/pages/branding/about.tsx

```diff
@@ -17,6 +17,109 @@ import OpenCollectiveIcon from 'docs/src/modules/branding/icons/OpenCollective';
 import BrandingRoot from 'docs/src/modules/branding/BrandingRoot';
 import BrandingBeginToday from 'docs/src/modules/branding/BrandingBeginToday';
 import BrandingDiscoverMore from 'docs/src/modules/branding/BrandingDiscoverMore';
+import BrandingPersona from 'docs/src/modules/branding/BrandingPersona';
+
+const emeriti = [
+  {
+    name: 'Hai Nguyen',
+    github: 'hai-cea',
+    twitter: 'haicea',
+    flag: 'v0.x creator',
+    location: 'Dallas, Texas, US',
+  },
+  {
+    name: 'Nathan Marks',
+    github: 'nathanmarks',
+    flag: 'v1.x co-creator',
+    location: 'Toronto, ON',
+  },
+  {
+    name: 'Kevin Ross',
+    github: 'rosskevin',
+    twitter: 'rosskevin',
+    flag: 'Core focus, flow',
+    location: 'Franklin, Tennessee, US',
+  },
+  {
+    name: 'Sebastian Sebald',
+    github: 'sebald',
+    twitter: 'sebastiansebald',
+    flag: 'Core focus',
+    location: 'Freiburg, Germany',
+  },
+  {
+    name: 'Ken Gregory',
+    github: 'kgregory',
+    flag: 'Core focus',
+    location: 'New Jersey, US',
+  },
+  {
+    name: 'Tom Crockett',
+    github: 'pelotom',
+    twitter: 'pelotom',
+    flag: 'Core focus',
+    location: 'Los Angeles, California, US',
+  },
+  {
+    name: 'Maik Marschner',
+    github: 'leMaik',
+    twitter: 'leMaikOfficial',
+    flag: 'Core focus',
+    location: 'Hannover, Germany',
+  },
+  {
+    name: 'Oleg Slobodskoi',
+    github: 'kof',
+    twitter: 'oleg008',
+    flag: 'JSS',
+    location: 'Berlin, Germany',
+  },
+  {
+    name: 'Dmitriy Kovalenko',
+    github: 'dmtrKovalenko',
+    twitter: 'dmtrKovalenko',
+    flag: 'Date pickers',
+    location: 'Kharkiv, Ukraine',
+  },
+  {
+    name: 'Josh Wooding',
+    github: 'joshwooding',
+    twitter: 'JoshWooding_',
+    flag: 'Core focus, J.P. Morgan',
+    location: 'London, UK',
+  },
+];
+
+function BrandingEmeriti() {
+  return (
+    <Box sx={{ bgcolor: 'greyEA', pt: 10, pb: 9 }}>
+      <Container>
+        <Typography variant="h3" component="h2">
+          Community <UnderlinedText>Emeriti</UnderlinedText>
+        </Typography>
+        <Typography sx={{ mt: 1.5, mb: 7, maxWidth: 700 }}>
+          We honor some no-longer-active core team members who have made valuable contributons in
+          the past. They advise us from time-to-time.
+        </Typography>
+        <Grid container spacing={4}>
+          {emeriti.map((member) => (
+            <Grid item xs={12} sm={6} md={4} lg={3} key={member.name}>
+              <BrandingPersona
+                name={member.name}
+                src={`https://github.com/${member.github}.png`}
+                title={member.flag}
```

Comment:
```suggestion
                title={member.title}
```
or rename both to `flag` :)